pkg/scheduler/framework/runtime/framework.go

```diff
@@ -673,7 +674,16 @@ func (f *frameworkImpl) RunPreFilterPlugins(ctx context.Context, state *framewor
 		if !s.IsSuccess() {
 			s.SetPlugin(pl.Name())
 			if s.IsRejected() {
```

Comment:
There is little value in calling this function, because every branch inside here looks different. Just compare the codes directly.